SU-HSDO/suhumsci
# SHS-5567: Implementation: Hide "Show as Expanded" toggle in menu settings on node form

# READY FOR REVIEW

## Summary
Menu Link toggle `Show as Expanded` is hidden on node edit/add pages. Menu link functionality remains the same.

## Steps to Test
1. Create a new flexible page
2. Open the menu link side menu by clicking on the `Provide Menu Link` checkbox 
3. Verify `Show as Expanded` is not displayed to user
4. Verify newly created node menu links behave the same

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
